spec/asciidoctor-standoc/refs_spec.rb

```diff
@@ -1,6 +1,7 @@
 require "spec_helper"
 require "relaton_iso"
 require "relaton_ietf"
+require "algolia/error"
```

Comment:
We should catch the `Algolia` error inside the `relaton` gem, such as under `RelatonBib::ConnectionError`: `Metanorma::StanDoc` should not need to know about the `algolia` gem at all.

Reply:
Fixed

Reply:
@CAMOBAP795 can you help point out the fix? Thanks!

Reply:
Test update to not use Algolia related exception

Reply:
Want to remove Algolia here.

Reply:
My bad lost them during resolving conflicts with master, will do it

Reply:
@ronaldtse fixed